src/main/java/practice/StreamPractice.java

```diff
@@ -34,8 +47,12 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * <p>
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
+
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getSex() == Person.Sex.MAN)
+                .filter(person -> person.getAge() >= fromAge && person.getAge() <= toAge)
```

Comment:
![image](https://github.com/mate-academy/jv-stream-github-practice/assets/20326858/2ac859ec-0022-48c2-a328-90cdbab84d6b)

```suggestion
                .filter(p -> p.getSex() == People.Sex.MAN && p.getAge() >= fromAge
                    && p.getAge() <= toAge)
```